static/styles/popovers.scss

```diff
@@ -194,6 +194,11 @@ ul.remind_me_popover .remind_icon {
     font-size: 15px;
 }
 
+.user-profile-modal-field-value[data-type="2"] {
+    white-space: pre-line;
+    overflow-wrap: break-word;
+}
```

Comment:
This will change to `.user-profile-key-value-section[data-type="2"] .user-profile-modal-field-value {`